### ISSUE TITLE
Constant Value for Pipeline Aggregation

### ISSUE THREAD

**Issue by jfeldman325** [raffis/mongodb-query-exporter#198]
#### Is your feature request related to a problem? Please describe
Very often in order to avoid blowing up prometheus metrics affinity you want to utilize an info metrics which contains the majority of labels which are joined on by other metrics. Decent summary [here](https://grafana.com/blog/2021/08/04/how-to-use-promql-joins-for-more-effective-queries-of-prometheus-metrics-at-scale/), at the moment there is no documented way of providing a constant value instead of selecting the value from the result

#### Describe the solution you'd like
A direct way to set a fixed value, for example:
```
  - name:  job_info
      type: constant
      help: "Deadline job information"
      value: 1
      labels: ["_id","group","pool","priority","user","batch","name"]
```

#### Describe alternatives you've considered
It seems possible to create a seperate aggregation which returns a fixed value field, but that seems like a hack overall.

**Comment by raffis** [comment]
It is actually possible to define constant labels by using `constLabels`. However at this moment it's only possible for metrics which are also bound to an aggregation. I never considered somebody using that without actually running an aggregation.
Not sure if this really makes sense but would be an easy addition.

**Comment by jfeldman325** [comment]
> > Yeah I can see the constant labels being useful in some situations, this is more about constant values which as mentioned can be highly useful for generating metrics primarily for joining! We (CoreWeave) have some very large prometheus metrics which simply wouldn't be usable without them.
> 
> You may try setting this, this should still export a metric if there was no value with the value of `1`, although your workaround is not necessarily that bad, just adds a tiny bit more load to the aggregation execution.
> 
> ```
>     overrideEmpty: true
>     emptyValue: 1
> ```

I tried this but thought it was only if nothing was returned from the aggregation. What would I set the `value` field to? From one aggreation i'm exporting a bunch of metrics with low label affinity and then want to expose a single high affinity metrics as described.

**Comment by raffis** [comment]
Yes you are right this wont work in its current state. But I think this should be possible to support. The proposal is to leave value empty in that case.

**Comment by raffis** [comment]
This should now be included with v5.1.0.

**Comment by jfeldman325** [comment]
> This should now be included with v5.1.0.

Amazing, thank you!
